.github/workflows/ci.yml

```diff
@@ -10,25 +12,22 @@ on:
     paths-ignore: '**/*.md'
 
 jobs:
-  test:
-    runs-on: ubuntu-20.04 # aka focal, because this is what we use in Travis
+  build:
+    runs-on: ubuntu-20.04  # aka focal, because this is what we use in Travis
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 10
```

Comment:
Do we need `fetch-depth: 0` maybe for license plugin?

Reply:
this one doesn't use the license plugin, but it is a good point. we should do that for copy/pasta defense